### ISSUE TITLE
finetuning, fc6 layer initialization

### ISSUE THREAD

**Issue by westpilgrim** [wy1iu/sphereface#33]
first , i train my model using original softmax
then, i want to finetune it using a-softmax
i intend to initialize the a-softmax's fc6 layer param using the original softmax, but the log says that they don't have the same number of param blobs, I found that margin inner product layer has no bias param blob.
i tried xavier initialization , but when i freeze the layers before fc6 and only learn fc6, the loss cannot converge .Do i have to relax the lr of the layers before fc6?
